webapp/channels/src/components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -1135,6 +1140,16 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
         const messageIsEmpty = this.state.message.length === 0;
         const draftMessageIsEmpty = this.props.draft.message.length === 0;
 
+        // fix for FF not capturing the paste without formatting event when using ctrl|cmd + shift + v
+        if (e.key === KeyCodes.V[0] && e.metaKey) {
```

Comment:
This should probably include `e.ctrlKey` as well. If you move this down a couple lines, you'll have `ctrlOrMetaKeyPressed` which you can use for that.

Or you can use the `Keyboard.cmdOrCtrlPressed` utility function which is also used below. We have too many ways of checking that 😛